PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,26 @@
-## JIRA Link
+### JIRA Link
 
+[Link](https://project44.atlassian.net/browse/XXXX)
 
-## Summary of Change
+### Summary
+Please include a summary of the change and add relevant motivation and context. Keep in mind that this information might be used by on-call engineers to handle potential incidents.
 
+### Risks
 
-## Proof
-*Add images of UI, an API response, RFC, Dashboards, or other context that you used or referenced that can help others understand and see the change.*
+- Potentially, what could go wrong with this change in the worst case scenario (i.e. not being able to process write requests, etc.)?
+- Please DO fill this in. The only change without the risk is no change at all.
 
+### Test Plan
```

Comment:
I really don't like this, thinking we have to define a test plan per PR. This should be a default part of the service runbook. Specific notes to the deployer is fine but this seems like too much process

Reply:
The point of this section is to give a rough idea on how can I validate that this change works if an original developer is unavailable and to get an idea on what user path it touches. It doesn't have to be a formal E2E test definition, just a simple `do this -> open that page in Movement -> see that something happened`.

While having service runbook is a good idea in general, this section should just refer to a single concrete thing that your change touches.